app/helpers/application_helper.rb

```diff
@@ -276,4 +276,10 @@ def sign_in_path
   def unread_notifications(user)
     Comment.accessible_by(current_ability).find_since_last_login(user)
   end
+
+  def any_event_date?(events, date)
+    date_ini =  DateTime.parse("#{date} 00:00")
+    date_fin =  date_ini + 1.day
```

Comment:
In the case of `date_ini` to be able to do `date_ini + 1.day` instead of parsing the date again. In the case of `date_fin` I thought it was more clear. I'll delete `date_fin.`